docs/api.md

```diff
@@ -62,6 +65,27 @@ python <script_path> train --help
 python <script_path> predict --help
 ```
 
+__Using Python code__
```

Comment:
Maybe Using Python script would be more appropriate? (not sure)

Reply:
It feels a bit better that python code, I'll update the wording !